src/machine/board_circuitplay_express.go

```diff
@@ -0,0 +1,70 @@
+// +build sam,atsamd21,circuitplay_express
+
+package machine
+
+// GPIO Pins
+const (
+	D0  = PB09
+	D1  = PB08
+	D2  = PB02
+	D3  = PB03
+	D4  = PA28
+	D5  = PA14
+	D6  = PA05
+	D7  = PA15
+	D8  = PB23
+	D9  = PA06
+	D10 = PA07
+	D11 = 0xff // does not seem to exist
+	D12 = PA02
+	D13 = PA17 // PWM available
+)
+
+// Analog Pins
+const (
+	A0  = PA02 // PWM available, also ADC/AIN[0]
+	A1  = PA05 // ADC/AIN[5]
+	A2  = PA06 // PWM available, also ADC/AIN[6]
+	A3  = PA07 // PWM available, also ADC/AIN[7]
+	A4  = PB03 // PORTB
+	A5  = PB02 // PORTB
+	A6  = PB09 // PORTB
+	A7  = PB08 // PORTB
+	A8  = PA11 // ADC/AIN[19]
+	A9  = PA09 // ADC/AIN[17]
+	A10 = PA04
+)
+
+const (
+	LED       = D13
+	NEOPIXELS = D8
```

Comment:
I know these are often called "neopixels", but that's a brand name, not a chip name. We are not Adafruit so I think it's better to stick with the chip name (WS2812 or WS2812B), also for other boards that might also expose such an array of LEDs.

Reply:
I named this constant to try to match the unique characteristic of the board, so naive developers would have an easier time figuring out how to use it. I do not think very many boards (read none) have a WS2812 built-in, except for these Adafruit boards.

Reply:
Ok, we can always add an alias in the future if needed.